client/packages/purchasing/src/common/Inputs.tsx

```diff
@@ -186,6 +174,7 @@ const NumInputRow = ({
             max={max}
             decimalLimit={decimalLimit ?? 0}
             endAdornment={endAdornment}
+            {...rest}
```

Comment:
Send 'em on down...

(The use case here was "autofocus", which I used to make the "Requested" field automatically focus on load)